tests/test_scale_properties.py

```diff
@@ -1,66 +1,94 @@
+    scale = persisted_scale.load()
+
+    coeff_list = [1.0, 2.0]
+    scale.poly_forward_coeff = coeff_list
+
+    assert scale.poly_forward_coeff == coeff_list
+
+
+def test__scale__get_Invalid_float64_list_property__throws_DaqError():
+    """Test for validating getter for invalid float list property in linear scale."""
+    linear_scale = Scale.create_lin_scale("custom_linear_scale", 5)
+    try:
+        _ = linear_scale.poly_forward_coeff
+    except DaqError as e:
+        assert e.error_code == -200601
+
 
-    def test_float64_property(self):
-        """Test for validating float property."""
-        scale = Scale("double_gain_scale")
+def test__scale__enum_property__returns_default_value():
+    """Test for validating getter for enum property."""
+    scale = Scale.create_lin_scale(
+        "custom_linear_scale", 5, y_intercept=1, pre_scaled_units=UnitsPreScaled.VOLTS
+    )
 
-        # Test property default value.
-        default_value = scale.lin_slope
-        assert scale.lin_slope == default_value
+    assert scale.pre_scaled_units == UnitsPreScaled.VOLTS
 
-        # Test property setter and getter.
-        scale.lin_slope = 5
-        assert scale.lin_slope == 5
 
-    def test_list_float64_property(self):
-        """Test for validating float list property."""
-        scale = Scale("polynomial_scale")
+def test__scale__set_enum_property__returns_assigned_value():
+    """Test for validating setter for enum property."""
+    scale = Scale.create_lin_scale(
+        "custom_linear_scale", 5, y_intercept=1, pre_scaled_units=UnitsPreScaled.VOLTS
+    )
 
-        # Test property default value.
-        default_value = scale.poly_forward_coeff
-        assert scale.poly_forward_coeff == default_value
+    scale.pre_scaled_units = UnitsPreScaled.AMPS
 
-        # Test property setter and getter.
-        coeff_list = [1.0, 2.0]
-        scale.poly_forward_coeff = coeff_list
-        assert scale.poly_forward_coeff == coeff_list
+    assert scale.pre_scaled_units == UnitsPreScaled.AMPS
 
-        # Test Incorrect property usage. Reading this property value throws DaqError.
-        linear_scale = Scale.create_lin_scale("custom_linear_scale", 5)
-        try:
-            _ = linear_scale.poly_forward_coeff
-        except DaqError as e:
-            assert e.error_code == -200601
 
-    def test_enum_property(self):
-        """Test for validating enum property."""
-        scale = Scale.create_lin_scale(
-            "custom_linear_scale", 5, y_intercept=1, pre_scaled_units=UnitsPreScaled.VOLTS
-        )
+@pytest.mark.parametrize("persisted_scale", ["double_gain_scale"], indirect=True)
+def test__scale__string_property__returns_default_value(persisted_scale):
+    """Test for validating getter for string property."""
+    scale = persisted_scale.load()
 
-        # Test property default value.
-        assert scale.pre_scaled_units == UnitsPreScaled.VOLTS
+    assert scale.description == ""
```

Comment:
Please test a more interesting property, or set the persisted `Description` to something more interesting.

Reply:
Instead of getting `description` string property, I created a custom `polynomial_scale` and read the `scaled_units` property value from the create scale.

Reply:
Okay, but it's no longer a persisted scale then, so update the test name.